### PR TITLE
Donations Module

### PR DESCRIPTION
**Pull request by ibanner**
Following recent correspondence, this pull request now includes:

- RTL style
- Hebrew strings (.po/.mo)
- Responsive styling
- Placeholder function for subscribing donors via ActiveTrail API (see below).

## INSTALLATION
The module relies on creating a few pages, and setting them up with the right page templates. Given the scope of the project I saw no reason to perform this setup programmaticly. 

I guess it would be best if you could pull this into your test environment and I'll take care of the necessary setup. If you prefer, I can document the necessary setup procedure for you guys to perform. There's also a dedicated options sub-page that needs to be setup.

### Donor Newsletter Subscription
The code in `newsletter-api.php` appears to be using a stale version of the API. I wrote the function based on the [Current API](http://webapi.mymarketing.co.il/api/docs) but actual API credentials & settings should be added.

### iCount Integration
I've set up 2 billing forms under BH's iCount new donations account. One is for one-time donations and the other for recurring ones. The module includes a `README.MD` file that includes the custom CSS I've used to style the forms in order to fit info our iframes. 

There are still some loose ends to be picked up in this integration:
@nuritgazit / @Netush - 
- [ ] Choose the template for the receipt sent to the donor
- [ ] Design and create the template of the email containing the receipt. 

@TheGrandVizier  - 
- [ ] Once we're ready for launch, we need to ask iCount to set our billing form to live mode

### PDF Certificates
This is not supported at the moment, as the tests I made produced a low quality results. I have an idea on how we achieve better results (TBD) but I thought best to create the pull request now, and work on this while you review the bulk of the module.

### CODE REVIEW

**Comment by ibanner** [comment]
**Update:** Following a phone call with @nuritgazit, the decision is to drop the automatic PDF creation for the time being. Upon successful CC transaction, an email will be sent (recipient TBD) with instructions on the texts to be integrated into the template. Currently I've setup a webhook in Zapier that sends an email with the relevant data. I can either edit this webhook (and move it to BH's Zapier account, if you have one) or implement it in the module itself. @nirgoldberg / @TheGrandVizier - any preference?

**Comment by nirgoldberg** [comment]
@ibanner, We don't have Zapier account, so if it's not complicated, please implement it within the module.

**Comment by ibanner** [comment]
@nirgoldberg No worries, will do. Could I suggest you pull this in first, as a topic branch, and I'll push my commits to it?

File: wp-content/themes/bh/views/donate/README.MD [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# Donation Pages
```
Comment (nirgoldberg) [inline wp-content/themes/bh/views/donate/README.MD]: What's this file's purpose?

File: wp-content/themes/bh/views/donate/thank-you.php [hunk 2]
```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
```
Comment (nirgoldberg) [inline wp-content/themes/bh/views/donate/thank-you.php]: Has this page defined outside the theme template, on purpose?
Is there a way to go back to the website after a donation has placed?

File: wp-content/themes/bh/views/donate/README.MD (hunk 1 above)
Comment (ibanner) [inline wp-content/themes/bh/views/donate/README.MD]: Merely to document/backup the custom CSS code that was written for the iCount forms (the iframes)

File: wp-content/themes/bh/views/donate/thank-you.php (hunk 2 above)
Comment (ibanner) [inline wp-content/themes/bh/views/donate/thank-you.php]: This file is meant to be shown inside the iframe (redirected from iCount after successful transaction). Therefore I had to "strip" the header/footer (we're staying inside the template all along).

File: wp-content/themes/bh/scss/donate.scss [hunk 3]
```diff
@@ -0,0 +1,644 @@
...
+  height: 620px;
+}
+
+/**************/
+/* responsive */
+/**************/
+
+@import "responsive";
+
+@media (max-width: 1199px) {
```
Comment (ibanner) [inline wp-content/themes/bh/scss/donate.scss]: Done!

**Comment by ibanner** [comment]
@nirgoldberg I think I took care or all your comments + added the email function that will replace the Zapier webhook.

File: wp-content/themes/bh/scss/donate.scss [hunk 4]
```diff
@@ -400,104 +415,83 @@ h1 {
...
 /**************/
 /* responsive */
 /**************/
 
-@import "responsive";
-
-@media (max-width: 1199px) {
-
-}
+// @import "responsive";
```
Comment (nirgoldberg) [inline wp-content/themes/bh/scss/donate.scss]: Something doesn't seem OK here: although you moved all responsive data to _responsive.scss, you still don't access _responsive.scss as you commented this line.
In addition, please remove all commented definitions below this line.

File: wp-content/themes/bh/languages/he_IL.po [hunk 5]
```diff
@@ -2,20 +2,19 @@ msgid ""
...
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 2.0.2\n"
+"X-Generator: Poedit 2.0.3\n"
 "X-Poedit-Basepath: ..\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "X-Poedit-KeywordsList: __;_e;_n:1,2;_x:1,2c;_ex:1,2c;esc_html__;esc_html_e;"
 "esc_html_x:1,2c\n"
 "X-Poedit-SourceCharset: UTF-8\n"
-"X-Poedit-Flags-xgettext: --add-comments\n"
```
[the remark this answers is not in the document]
Comment (ibanner) [reply wp-content/themes/bh/languages/he_IL.po]: @nirgoldberg I'm not sure what's still wrong here... The line with the `X-Poedit-Flags-xgettext` was removed in my last commit...

**Pull request merged**